docs/source/install/installation.rst

```diff
@@ -294,144 +294,21 @@ If you open a new terminal for example, be sure to activate it first with::
 At this point, you can choose to read on for additional installation details and configuration options, or you can choose to start using
 AiiDA and go straight to the section :ref:`get started<get_started>`.
 
+.. _start_daemon:
 
-.. _configure_aiida:
+Start the daemon
+================
 
-Configure AiiDA
-===============
+The AiiDA daemon process runs in the background and takes care of
+submitting new calculations, checking their status, retrieving their results once they are finished and storing them in the AiiDA database.
```

Comment:
This is not strictly true, for two reasons:
* it will also run work chains that are submitted
* it is not really only the daemon that can deal with calculation jobs. You can also run a calculation job, in which case a local runner will do all the things listed here.

Reply:
thanks - my rewrite is hopefully better